index.js

```diff
@@ -1329,6 +1340,48 @@ function pad(str, width) {
   return str + Array(len + 1).join(' ');
 }
 
+/**
+ * Wraps the given string with line breaks at the specified width while breaking
+ * words and indenting every but the first line on the left.
+ *
+ * @param {String} str
+ * @param {Number} width
+ * @param {Number} indent
+ * @return {String}
+ * @api private
+ */
+function wrap(str, width, indent) {
+  var regex = new RegExp('.{1,' + (width - 1) + '}([\\s\u200B]|$)|[^\\s\u200B]+?([\\s\u200B]|$)', 'g');
+  var lines = str.match(regex) || [];
+  var result = lines.map(function(line, i) {
```

Comment:
How about returning directly without assigning to a variable?

Reply:
Done